stdlib/public/runtime/ProtocolConformance.cpp

```diff
@@ -242,11 +242,10 @@ static void _addImageProtocolConformancesBlock(const uint8_t *conformances,
 struct ConformanceState {
   ConcurrentMap<ConformanceCacheEntry> Cache;
   std::vector<ConformanceSection> SectionsToScan;
-  pthread_mutex_t SectionsToScanLock;
+  RecursiveMutex SectionsToScanLock;
```

Comment:
Nope it doesn't need to be recursive. I was using this file to poke at RecursiveMutex but looks like I missed reverting this. :disappointed: